test/core/linter_test.py

```diff
@@ -623,6 +667,9 @@ def test_linter_noqa():
         col_q q, --Inline comment --noqa: L012
         col_r r, /* Block comment */ --noqa: L012
         col_s s # hash comment --noqa: L012
+        , col_t t --noqa: L01*
+        , col_u u -- Some comment --noqa: L01*
+        , col_v v -- We can ignore both L012 and L019 -- noqa: L01[29]
```

Comment:
Why are we mixing trailing and leading commas here?

Reply:
This is so it triggers both L012 and L019, otherwise we're not really testing that the glob matching ignores multiple rules.

Reply:
Ah that wasn’t obvious to me. Could we expand the comment to note that? Or add another comment line above these three lines?

Reply:
Will do 👍